Amplify/DevMenu/Data/PluginInfoItem.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@available(iOS 13.0.0, *)
+struct PluginInfoItem: Identifiable, InfoItemProvider {
+
+    let id = UUID()
+    var displayName: String
+    var information: String
+
+    init(displayName: String, information: String) {
+        self.displayName = displayName.isEmpty ? "Unknown Plugin" : displayName
+        self.information = information.isEmpty ? "Not Available" : information
```

Comment:
Let us move the string constants to a separate file "DevMenuStringConstants". I see some strings reused like "Not Available". Example - https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Support/Constants/AuthPluginErrorConstants.swift